scripts/dev_tools/local_review.py

```
@@ -0,0 +1,243 @@
+    get_ollama_endpoint,
+    get_ollama_model,
+    validate_ollama_connection,
+)
+from review_common import build_prompt, truncate_diff
+
+
+def get_git_root() -> str:
+    """Get the root directory of the git repository."""
+    try:
+        result = subprocess.run(
+            ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)